docs/docs/rest-api/public/api/v2/types/pod.raml

```diff
@@ -0,0 +1,80 @@
+#%RAML 1.0 DataType
+type: object
+description:|
+  A pod allows one to launch a collection co-located (on the same agent) containers
+  that share the same network namespace and that may share the same volumes.
+  Resources are specified on a per-container basis.
+properties:
+  id:
+    type: !include pathId.raml
+    description:|
+      Unique identifier for the pod consisting of a series of names separated by slashes.
+      Each name must be at least 1 character and may only contain digits (`0-9`), dashes
+      (`-`), dots (`.`), and lowercase letters (`a-z`). The name may not begin or end with a dash.
+    required: true
+    example: /ops/audit
+  containers:
+    type: array
+    items: !include mesosContainer.raml
+  instances:
+    type: integer
+    description: The number of instances of this pod to start.
+    minimum: 0
+  maxInstances?:
+    type: integer
+    description:|
+      The maximum number of instances of this pod that should
+      be running at a given time.
+    minimum: 0
+  constraints?:
+    type: array
+    items: !include constraint.raml
+    description:|
+      Valid constraint operators are one of UNIQUE, CLUSTER, GROUP_BY, LIKE, UNLIKE, MAX_PER
+    example: [hostname:UNIQUE]
+  user?:
+    type: string
+    description:|
+      The OS user to use to run the tasks on the agent.
+      May be overridden by a container.
+  version?:
+    type: datetime-only
```

Comment:
why is this a `datetime-only` type? shouldn't the nature of this data be opaque from an API perspective (could be a integer, timestamp, string, whatever -- the user should not care since they never set it)